bpm.sh

```diff
@@ -35,7 +35,7 @@ if ${BPM_LOADED:-false}; then
 else
     __bpm_info() { :; }
 fi
-shopt -s extglob
+shopt -s extglob nullglob
```

Comment:
```suggestion
shopt -s extglob
```
I'm not entirely comfortable with nullglob for all my sessions.  Is this mainly for the `ls -tdL ... ./*` below?  Then I'd suggest we turn it on for that command only.